src/tools/compiletest/src/runtest.rs

```diff
@@ -3590,7 +3653,7 @@ pub struct ProcRes {
 }
 
 impl ProcRes {
-    pub fn fatal(&self, err: Option<&str>) -> ! {
+    pub fn fatal(&self, err: Option<&str>, ctx: impl FnOnce()) -> ! {
```

Comment:
Isn't this some sort of callback on failure? It may not be so obvious `ctx` means that, I also didn't know `ctx` is a function.